{"errorMessage":"Lambda process returned error status code: 1. Result: {\"errorMessage\": \"Syntax error in module 'reads3'\"}. Output:\nSTART RequestId: 7a970ede-1f07-4c41-9394-90ec941c5543 Version: $LATEST\nSyntax error in module 'reads3': invalid syntax (reads3.py, line 10)\nEND RequestId: 7a970ede-1f07-4c41-9394-90ec941c5543\nREPORT RequestId: 7a970ede-1f07-4c41-9394-90ec941c5543 Duration: 0 ms Billed Duration: 100 ms Memory Size: 1536 MB Max Memory Used: 19 MB","errorType":"InvocationException","stackTrace":["  File \"/opt/code/localstack/localstack/services/awslambda/lambda_api.py\", line 574, in run_lambda\n    result = LAMBDA_EXECUTOR.execute(func_arn, func_details, event, context=context,\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 176, in execute\n    return do_execute()\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 168, in do_execute\n    return _run(func_arn=func_arn)\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 149, in wrapped\n    raise e\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 145, in wrapped\n    result = func(*args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 159, in _run\n    raise e\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 147, in _run\n    result = self._execute(func_arn, func_details, event, context, version)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 325, in _execute\n    result = self.run_lambda_executor(cmd, stdin, env_vars=environment, func_details=func_details)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 231, in run_lambda_executor\n    raise InvocationException('Lambda process returned error status code: %s. Result: %s. Output:\\n%s' %\n"]}
